doc/bridge_packages/flame_riverpod/riverpod.md

```diff
@@ -0,0 +1,67 @@
+# flame_riverpod
+
+
+## Riverpod
+
+[Riverpod](https://pub.dev/packages/flutter_riverpod) is a reactive caching and data-binding
```

Comment:
```suggestion
[Riverpod](https://riverpod.dev/) is a reactive caching and data-binding
```
Maybe linking to their page instead since there is more info there?